src/main/java/com/google/sps/servlets/RatingServlet.java

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.sps.data.Tutor;
+import com.google.sps.data.TimeRange;
+import com.google.sps.data.TutorSession;
+import com.google.sps.data.SampleData;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/rating")
+public class RatingServlet extends HttpServlet {
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        response.setContentType("plain/text");
+        response.getWriter().println("To be implemented");
+    }
+
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        String tutorEmail = request.getParameter("tutorEmail");
+        String studentEmail = request.getParameter("studentEmail");
+        int rating = Integer.parseInt(request.getParameter("rating"));
+
+        // Update tutor's rating
+        SampleData.rateTutorByEmail(tutorEmail, studentEmail, rating);
+
+        String jsonTutors = new Gson().toJson(SampleData.getSampleTutors());
+        String jsonStudents = new Gson().toJson(SampleData.getSampleStudents());
+        String json = jsonTutors + ", " + jsonStudents; 
```

Comment:
I wonder if this would work so you can send an array instead of just comma separated strings:
```suggestion
        String json = new Gson().toJson(new String[]{jsonTutors, jsonStudents}); 
```

Reply:
That worked! Thanks :)